src/gui/PageLoader.qml

```diff
@@ -61,6 +71,22 @@ HLoader {
         return true
     }
 
+    function moveThroughVisitHistory(relativeMovement=1) {
+
+        // not allowed to go beyond oldest entry in history
+        if (historyPosition + relativeMovement >= history.length) return false
+
+        // not allowed to go beyond newest entry in history
+        if (historyPosition + relativeMovement < 0) return false
```

Comment:
It would be more useful to wrap around here rather than just returning

Reply:
I should call `show` for switching to the room that's currently shown already?

Reply:
- If there's no entries in the history, just return
- if user goes back while already being at index 0, the new index should be the last history entry
- if user goes forward while being at last index, new index should be 0

Reply:
It comes down to personal preference. For me, I'd much rather have the button do nothing, as an indication that I'm at the end of the list. So I added a setting for this. See latest commit